ext/opcache/tests/preload_012.phpt

```diff
@@ -11,5 +11,16 @@ require_once('skipif.inc');
 if (PHP_OS_FAMILY == 'Windows') die('skip Preloading is not supported on Windows');
 ?>
 --FILE--
---EXPECTF--
-Warning: Can't preload class Test with unresolved initializer for constant C in %s on line %d
+<?php
```

Comment:
Same feedback as in `ext/opcache/tests/preload_loadable_classes_2.phpt`: would be better to keep `--FILE--` as it was, then change only `--EXPECTF--`

Reply:
We would have test with empty --FILE-- and empty --EXPECT-- . The new code shows, that now Test is preloaded, but resolution of constant "Test::C" failed, because "Foo::BAR" is not defined.